monticore-generator/src/main/java/de/monticore/codegen/cd2java/_symboltable/serialization/BITSer.java

```diff
@@ -49,6 +50,9 @@ public void init() {
     bitsers.put("long", _long);
     bitsers.put("java.lang.Long", _long);
     bitsers.put("Long", _long);
+
+    BITSerStrategy symTypeExpression = new SymTypeExpressionSerStrategy();
+    bitsers.put(SymTypeExpression.class.getName(), symTypeExpression);
```

Comment:
Honestly, I don't know, but probably not.
Would be future proofing, in case this ever gets changed, but I assume you are correct that it should work in all cases as of now